supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchRuntimeAttributes.scala

```diff
@@ -151,16 +151,34 @@ object AwsBatchRuntimeAttributes {
   }
 }
 
-object QueueArnValidation {
-  lazy val instance: RuntimeAttributesValidation[String] = ArnValidation(AwsBatchRuntimeAttributes.QueueArnKey)
-
-  def configDefaultWomValue(config: Option[Config]): Option[WomValue] = instance.configDefaultWomValue(config)
+object QueueArnValidation extends ArnValidation(AwsBatchRuntimeAttributes.QueueArnKey) {
+  // queue arn format can be found here
+  // https://docs.aws.amazon.com/en_us/general/latest/gr/aws-arns-and-namespaces.html#arn-syntax-batch
+  // arn:aws:batch:region:account-id:job-queue/queue-name
+  override protected val arnPattern = "(arn):(aws(-[a-z]+){0,2}):(batch):([a-z]{2}(-[a-z]+){1,2}-\\d):(\\d{12}):(job-queue)/([\\w-]{1,128})".r
 }
-// TODO: provide arn regexp validation here
+
 object ArnValidation {
   def apply(key: String): ArnValidation = new ArnValidation(key)
 }
-class ArnValidation(override val key: String) extends StringRuntimeAttributesValidation(key)
+
+class ArnValidation(override val key: String) extends StringRuntimeAttributesValidation(key) {
+  override protected def validateValue: PartialFunction[WomValue, ErrorOr[String]] = {
+    case WomString(s) => validateArn(s)
+  }
+
+  private def validateArn(possibleArn: String): ErrorOr[String] = {
+    possibleArn match {
+      case arnPattern(_@_*) => possibleArn.validNel
+      case _ => "ARN has invalid format".invalidNel
+    }
+  }
+
+  // Possible ARN formats can be found here
+  // https://docs.aws.amazon.com/en_us/general/latest/gr/aws-arns-and-namespaces.html
+  // This is quite vague regex, but it allows to support a lot of ARN formats
+  protected val arnPattern = "(arn):(aws(-[A-Za-z]+){0,2}):([\\w-]+):([A-Za-z]{2}(-[A-Za-z]+){1,2}-\\d)?:(\\d{12})?:(([\\w- ]+[:/])?(((/\\*)|([\\w- /.:]))+))([:/]\\w+)?".r
```

Comment:
That is really difficult to reason about this regular expression in such formatting. We have a good example of well formatted regular expression in the [`DockerImageIdentifier.scala`](https://github.com/broadinstitute/cromwell/blob/develop/dockerHashing/src/main/scala/cromwell/docker/DockerImageIdentifier.scala#L36-L60) class. Please, format it and provide with any documentation you can come up with=)

The same about the regular expression above.

Reply:
It took a while, but now it is done.